### ISSUE TITLE
errors when scheduling a post

### ISSUE THREAD

**Issue by paulbgtr** [paulbgtr/skeetsched#2]
- Warning: Missing `Description` or `aria-describedby={undefined}` for {DialogContent}. 
- DialogContent` requires a `DialogTitle` for the component to be accessible for screen reader users.
